docs/pages/switch.js

```diff
@@ -57,6 +57,196 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+            defaultCode={`
+<Flex direction="column" gap={2} width="100%" maxWidth={300}>
+  <Heading size="300">Tune your home feed</Heading>
+  <Text>Turn off any boards that you don't want us to use for your home feed recommendations</Text>
+  <Flex direction="column" gap={4}>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Label htmlFor="boardfood">
+          <Flex direction="column" gap={1}>
+            <Text>Food</Text>
+            <Text size="100">55 pins · 1 section</Text>
+          </Flex>
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => {}}
+        id="boardfood"
+        switched={false}
+      />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Label htmlFor="boardoutfits">
+          <Flex direction="column" gap={1}>
+            <Text>Outfits</Text>
+            <Text size="100">138 pins · 5 sections</Text>
+          </Flex>
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => {}}
+        id="boardoutfits"
+        switched={true}
+      />
+    </Flex>
+    <Flex alignItems="center">
+      <Flex.Item flex="grow">
+        <Label htmlFor="boardhomedecor">
+          <Flex direction="column" gap={1}>
+            <Text>Home Decor</Text>
+            <Text size="100">33 pins · 2 sections</Text>
+          </Flex>
+        </Label>
+      </Flex.Item>
+      <Switch
+        onChange={() => {}}
+        id="boardhomedecor"
+        switched={true}
+      />
+    </Flex>
+  </Flex>
+</Flex>
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Accessibility"
+        description={`
+    Switches should have labels that can be read by screen readers, and that can be clicked or tapped to make it easier for users to select and deselect. Therefore, make sure to supply the \`label\` prop. If that is not possible, make sure your stand-alone Label has an \`htmlFor\` prop that matches the \`id\` on the Switch. Test that the Switch and label are properly connected by clicking or tapping on the label and confirming that it activates the Switch next to it.
+
+      If Switch is labeled by content elsewhere on the page, or a more complex label is needed, the \`labelDisplay\` prop can be used to visually hide the label. In this case, it is still available to screen reader users, but will not appear visually on the screen. See the [Label visibility example](https://gestalt.netlify.app/checkbox#Label-visibility) for more detail.`}
+      >
+        <MainSection.Subsection
+          title="Keyboard navigation"
+          columns={2}
+          description={`
+    Switch has conventional keyboard support.
+    - Users relying on the keyboard expect to move focus to the Switch by using the tab key or shift+tab when moving backwards.
```

Comment:
Maybe add - Once focused, the Space key toggles the Switch

Reply:
Nice! Done!